flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/util/KinesisConfigUtil.java

```diff
@@ -181,6 +181,25 @@ public static Properties replaceDeprecatedProducerKeys(Properties configProps) {
 		return configProps;
 	}
 
+	public static Properties replaceDeprecatedConsumerKeys(Properties configProps) {
+		if (configProps.containsKey(ConsumerConfigConstants.STREAM_DESCRIBE_BACKOFF_BASE)) {
```

Comment:
Minor: this could be generalized by iterating over a map of oldkey -> newkey and I would also suggest to log a warning for deprecated keys